### PR TITLE
Add an index page and an api page

### PR DESCRIPTION
**Pull request by JakeChampion**
A future pull-request will add the url-builder page :-)

### CODE REVIEW

**Comment by JakeChampion** [comment]
localhost_8080_v3_api
localhost_8080_

File: views/api.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+	<div class="o-layout__header">
+		{{> header }}
+	</div>
+
+	<div class="o-layout__sidebar"></div>
+
+	<div class="o-layout__main o-layout-typography" data-o-component="o-syntax-highlight">
+		<h1 id="#api-reference">API Reference</h1>
+		<div>
+			<h2 id="get-v3-bundle-js)">JavaScript</h2>
```
Comment (notlee) [inline views/api.html]:
```suggestion
			<h2 id="get-v3-bundle-js">JavaScript</h2>
```

File: views/api.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+							<code>/v3/bundles/js</code>
+						</td>
+					</tr>
+					<tr>
+						<th scope="row">Querystring</th>
+						<td>
+							<dl>
+								<dt>
+									<code>
+										<var>modules</var>
```
Comment (notlee) [inline views/api.html]: Feature(kinda) request: Shall we change this to components?

File: views/api.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+									<p>A comma-separated list of Origami components in the format <code>name@version</code>. The <code>name</code> can be any Origami component and the <code>version</code> is any <abbr title="Semantic Versioning">SemVer</abbr> <a href="https://semver.org/">compatible</a> version.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>source</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Required</strong>
+									<p>The systemcode of the System that is is making ue of the Origami Build Service.</p>
```
Comment (notlee) [inline views/api.html]:
```suggestion
									<p>The systemcode of the system that is is making use of the Origami Build Service.</p>
```

File: views/api.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+									</code>
+								</dt>
+								<dd>
+									<strong>Internal use only</strong>
+									<p>Set to a normalised useragent string to have the JavaScript be compiled to the latest version of JavaScript that the useragent supports.</p>
+									<p>This is used internally in the service and is exposed to simplify debugging a response and to acheive a simpler caching workflow with our CDN.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>registry</var>
```
Comment (notlee) [inline views/api.html]: Why is this part of the api, does it make a difference to the build service user?

_Appreciate this PR is about docs, if you'd rather reply to this question elsewhere_

Comment (JakeChampion) [reply views/api.html]: https://github.com/Financial-Times/origami-build-service-v3/issues/32

File: views/api.html [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+									<code>
+										<var>brand</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Defaults to <code>master</code></strong>
+									<p><a href="https://origami.ft.com/docs/components/branding/#supported-brands">Documentation about branding</a></p>
+									<p>Set to <code>master</code> to have the CSS use FT.com style branding.</p>
+									<p>Set to <code>internal</code> to have the CSS use internal FT product style branding.</p>
+									<p>Set to <code>whitelabel</code> to have the CSS use no brand specific styling.</p>
```
Comment (gvonkoss) [inline views/api.html]:
```suggestion
									<p>Set to <code>whitelabel</code> to have the CSS use no brand-specific styling.</p>
```

File: views/partials/footer.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+			<div class="o-footer-services__container">
+				<div class="o-footer-services__wrapper o-footer-services__wrapper--legal">
+					<div class="o-footer-services__links">
+						<a href="http://help.ft.com/help/legal-privacy/cookies/">Cookies</a>
+						<a href="http://help.ft.com/help/legal-privacy/copyright/copyright-policy/">Copyright</a>
+						<a href="http://help.ft.com/help/legal-privacy/privacy/"
+							class="o-footer-services__bulletted-link">Privacy</a>
+						<a href="http://help.ft.com/help/legal-privacy/terms-conditions">Terms &amp; Conditions</a>
+					</div>
+					<p><span>&#xA9; THE FINANCIAL TIMES LTD 2018.</span> FT and &apos;Financial Times&apos; are
```
Comment (gvonkoss) [inline views/partials/footer.html]:
```suggestion
					<p><span>&#xA9; THE FINANCIAL TIMES LTD 2019.</span> FT and &apos;Financial Times&apos; are
```

File: views/api.html [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+									<p>A comma-separated list of Origami components in the format <code>name@version</code>. The <code>name</code> can be any Origami component and the <code>version</code> is any <abbr title="Semantic Versioning">SemVer</abbr> <a href="https://semver.org/">compatible</a> version.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>source</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Required</strong>
+									<p>The systemcode of the system that is is making use of the Origami Build Service.</p>
```
[the remark this answers is not in the document]
Comment (JakeChampion) [reply views/api.html]: I couldn't come up with a good link 😔 Do you know of one?

File: views/partials/footer.html (hunk 6 above)
Comment (JakeChampion) [reply views/partials/footer.html]: 🙈

File: views/api.html (hunk 2 above)
Comment (JakeChampion) [reply views/api.html]: I've filled an issue for this work #33

File: views/api.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (gvonkoss) [reply views/api.html]: No ): Rhys says that you can find information about the system code if you click on 'i' beside the word in bizops but 🤷‍♀

**Pull request merged**